dataset/models/tf/fcn.py

```diff
@@ -0,0 +1,84 @@
+"""Contains class for FCN"""
+import tensorflow as tf
+from .layers import conv_block
+from . import TFModel
+from . import VGG16
+
+class FCN(TFModel):
+    """FCN as TFModel
```

Comment:
Leave the full name in the doctsring - Fully Convolutional Network
And mention the original article